UPGRADING.md

```diff
@@ -4,6 +4,11 @@ Whilst experimenting with the API & benefiting from changes in Ember, we've had
 
 Here are the details on updating from previous versions.
 
+## From v2.0.0
+1. You will not longer be able to import `CanMixin` and `computed` as they will be removed. You can reproduce theirs behaviours using `can` service and its methods.
```

Comment:
This seems to be inaccurate as the computed macro still exists. It's just imported differently.

Reply:
Some old stuff :/